packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -207,8 +208,16 @@ export default class VList {
      * @param type Type of this list item, can be ListType.None
      */
     appendItem(node: Node, type: ListType) {
-        node = getTagOfNode(node) == 'LI' ? node : wrap(node, 'li');
-        this.items.push(type == ListType.None ? new VListItem(node) : new VListItem(node, type));
+        let newListNode = node;
+        const nodeTag = getTagOfNode(node);
+
+        if (nodeTag != 'LI' && nodeTag != 'TABLE') {
```

Comment:
For table, we still need to wrap it with LI, right?
Please add a test case for table.

Reply:
Yes, I forgot to add the else clause. I've resolved it